server/user-bootstrap/index.js

```diff
@@ -1,32 +1,32 @@
 var superagent = require( 'superagent' ),
-	cookie = require( 'cookie' ),
 	debug = require( 'debug' )( 'calypso:bootstrap' ),
 	crypto = require( 'crypto' );
 
 var config = require( 'config' ),
 	API_KEY = config( 'wpcom_calypso_rest_api_key' ),
 	userUtils = require( './shared-utils' ),
-
+	AUTH_COOKIE_NAME = 'wordpress_logged_in',
 	/**
 	* WordPress.com REST API /me endpoint.
 	*/
 	url = 'https://public-api.wordpress.com/rest/v1/me?meta=flags';
 
-module.exports = function( userCookie, callback ) {
+
+module.exports = function( authCookieValue, callback ) {
 	// create HTTP Request object
 	var req = superagent.get( url ),
-		hmac, cookies, hash;
+		hmac, hash;
+
+	if ( authCookieValue ) {
+		authCookieValue = decodeURIComponent( authCookieValue );
 
-	if ( userCookie ) {
 		hmac = crypto.createHmac( 'md5', API_KEY );
-		cookies = cookie.parse( userCookie );
-		if ( cookies.wordpress_logged_in ) {
-			hmac.update( cookies.wordpress_logged_in );
-			hash = hmac.digest( 'hex' );
-			req.set( 'Authorization', 'X-WPCALYPSO ' + hash );
-			req.set( 'Cookie', userCookie );
-			req.set( 'User-Agent', 'WordPress.com Calypso' );
-		}
+		hmac.update( authCookieValue );
+		hash = hmac.digest( 'hex' );
+
+		req.set( 'Authorization', 'X-WPCALYPSO ' + hash );
+		req.set( 'Cookie', AUTH_COOKIE_NAME + '=' + authCookieValue );
+		req.set( 'User-Agent', 'WordPress.com Calypso' );
```

Comment:
What motivated/necessitated these changes?

Reply:
@stephanethomas motivated me to simplify that code here:
https://github.com/Automattic/wp-calypso/pull/18264#discussion_r141048376

We can also leave it as is and pass:
```js
user( `wordpress_logged_in=${ req.cookies.wordpress_logged_in }`, ...
```